qcodes/instrument_drivers/Lakeshore/Model_336.py

```diff
@@ -1,57 +1,48 @@
-from qcodes import VisaInstrument, InstrumentChannel, ChannelList
-from qcodes.utils.validators import Enum, Strings
-
-class SensorChannel(InstrumentChannel):
-    """
-    A single sensor channel of a temperature controller
-    """
+from typing import ClassVar, Dict
+import logging
+from .lakeshore_base import LakeshoreBase, BaseOutput, BaseSensorChannel
+import qcodes.utils.validators as vals
+
+log = logging.getLogger(__name__)
+
+class Output_336(BaseOutput):
+
```

Comment:
Can we add a docstring describing the usefulness of this class? This is not clear to me.

Reply:
done